shared/src/main/scala/tastyquery/ast/Symbols.scala

```diff
@@ -181,21 +181,15 @@ object Symbols {
         .takeWhile(s => s.maybeOuter.exists)
         .forall(s => s.isPackage || s.isClass && s.name.toTypeName.wrapsObjectName)
 
-    private def nameWithPrefix(addPrefix: Symbol => Boolean): Name =
-      if isPackage then name
+    private def nameWithPrefix(addPrefix: Symbol => Boolean): FullyQualifiedName =
+      if isPackage && (isRoot || owner.isRoot) then FullyQualifiedName(name :: Nil)
```

Comment:
I don't think we should be able to store the root package name in a fully qualified name, FQN is relative to the root

Reply:
this should crash for the root package, or allow empty FQN